grails-app/services/org/bbop/apollo/authenticator/RemoteUserAuthenticatorService.groovy

```diff
@@ -61,6 +64,22 @@ class RemoteUserAuthenticatorService implements AuthenticatorService {
                 log.debug "adding role: ${role}"
                 user.addToRoles(role)
                 role.addToUsers(user)
+
+                if (this.default_group != null) {
+                    log.debug "adding user to default group: ${this.default_group}"
+                    UserGroup userGroup = UserGroup.findByName(this.default_group)
+
+                    if (userGroup == null) {
```

Comment:
I like to use the elvis syntax here, but this is fine as well. 

userGroup = userGroup ?: new UserGroup (...).save(...)